app/packages/addons-service/provider.ts

```diff
@@ -216,9 +214,8 @@ export default class Provider {
             initiateOauth: initiateOauth || false,
             apiBaseUrl: (this.provider as ExternalStorageServiceModel).configurableApiRoot ? credentials.url : '',
             externalUserId: this.currentUser.user?.id,
-            scopes: [],
-            storageProvider: this.provider,
-            configuringUser: this.userReference,
```

Comment:
@brianjgeiger it looks like the removal of this line is what's messing up the user settings page. should add `accountOwner: this.userReference` here